client/web/compose/src/components/PageBlocks/RecordBase.vue

```diff
@@ -224,3 +296,25 @@ export default {
   min-height: 1.2rem;
 }
 </style>
+
+<style lang="scss" scoped>
+.field-label{
```

Comment:
You can do inline styles in template for this (d-flex align-items-center), the gap you dont need since you have ml-1 on the inline action anyway